README.md

```diff
@@ -43,7 +43,7 @@ App consists of 4 main parts which should cover whole app flow:
 - Manage my booking
 - Payments
 
-More detailed info can be found in [Features and Roadmap docs](https://kiwicom.github.io/margarita/docs/features-roadmap).
+More detailed info can be found in [Features docs](https://kiwicom.github.io/margarita/docs/features).
```

Comment:
in the [Features docs] (same  for every citation of the docs)

Reply:
Done